### PR TITLE
adding Fon language method

### PR DESCRIPTION
**Pull request by Mahuton**

### CODE REVIEW

File: rules/fon/fon-alt.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+
+( function ( $ ) {
+	'use strict';
+
+	var fonAlt = {
+		id: 'fon-alt',
+		name: 'fon-alt',
+		description: 'fon input keyboard',
+		date: '2018-05-18',
+		URL: 'http://github.com/wikimedia/jquery.ime',
```
Comment (kartikm) [inline rules/fon/fon-alt.js]: Minor: Use https

**Comment by kartikm** [comment]
It would be nice to have tests for this input method. See: https://github.com/wikimedia/jquery.ime/tree/master/test You can follow some existing tests.

File: rules/fon/fon-alt.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+		id: 'fon-alt',
+		name: 'fon-alt',
+		description: 'Fon input keyboard',
+		date: '2018-05-18',
+		URL: 'http://github.com/wikimedia/jquery.ime',
+		author: 'Mahuton POSSOUPE',
+		license: 'GPLv3',
+		version: '1.0',
+		patterns: [
+<<<<<<< HEAD
```
Comment (santhoshtr) [inline rules/fon/fon-alt.js]: please fix these merge conflict markers

Comment (amire80) [reply rules/fon/fon-alt.js]: I already fixed this with @Mahuton on his laptop. He just needs to upload the new version :)

File: test/jquery.ime.test.fixtures.js [hunk 3]
```diff
@@ -4127,5 +4127,17 @@ var palochkaVariants = {
 			{ input: [ 'v', [ '1', true ], 'i', 'T', 'U' ], output: 'វ‌ិទូ', description: 'ZWNJ (savant): v AltGr-1 iTU -> វ‌ិទូ' },
 			{ input: [ 'v', [ '`', true ], 'i', 'T', 'U' ], output: 'វ‍ិទូ', description: 'ZWJ (savant): v AltGr-` iTU -> វ‍ិទូ' }
 		]
+	},
+	{
+		description: 'fon alt test',
```
Comment (santhoshtr) [inline test/jquery.ime.test.fixtures.js]: I see that you renamed the input method to fon tilde, should not we change this description too?

File: test/jquery.ime.test.fixtures.js [hunk 4]
```diff
@@ -4127,5 +4127,17 @@ var palochkaVariants = {
...
+		tests: [
+			{ input: '~D', output: 'Ɖ', description: 'fon ~D -> Ɖ' },
+			{ input: '~d', output:'ɖ' , description: 'fon ~d -> ɖ' },
+			{ input: '~E', output: 'Ɛ', description: 'fon ~E -> Ɛ' },
+			{ input: '~e', output: 'ɛ', description: 'fon ~e -> ɛ '},
+			{ input: '~O', output: 'Ɔ', description: 'fon ~O -> Ɔ' },
+			{ input: '~o', output: 'ɔ', description: 'fon ~o -> ɔ' }
+		],
+		inputmethod: 'fonTilde'
```
Comment (santhoshtr) [inline test/jquery.ime.test.fixtures.js]: This should be the id of the input method fon-tilde in our case

**Pull request merged**